applications/nightly_build/test_efn.py

```diff
@@ -26,6 +25,7 @@ def tearDown(self):
         for fl in self.model_files:
             os.remove(fl)
 
+    @unittest.skip("TODO")
```

Comment:
Can you turn this case to tf.keras and validate it as well?

Reply:
Good point, turn to tf.keras, set rtol=1e-2 atol=5e-4 seems ok.